src/lib/components/sidebar/stories/SidebarItem.stories.svelte

```diff
@@ -32,6 +32,12 @@
   </SidebarItem>
 </Story>
 
+<Story name="Active">
+  <SidebarItem active>
+    <Home16 /> Go Home
+  </SidebarItem>
+</Story>
+
```

Comment:
Can delete this since we have another "Active" story below it

Reply:
Done